keyboards/rationalist/ratio65_solder/rev_a/rev_a.c

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2022 Stefan Sundin "4pplet" <mail@4pplet.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "rev_a.h"
+
+#define LED_PIN C4
+
+void matrix_init_kb(void) {
+    setPinOutput(LED_PIN);
+    matrix_init_user();
+}
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if(res) {
+        writePin(LED_PIN, !led_state.caps_lock);
+    }
+    return res;
+}
```

Comment:
Should use `LED_CAPS_LOCK_PIN` - <https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options>
```suggestion
```